src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Formatting/Passes/New/HtmlFormattingPass.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Razor.Language;
+using Microsoft.AspNetCore.Razor.Language.Syntax;
+using Microsoft.AspNetCore.Razor.PooledObjects;
+using Microsoft.CodeAnalysis.Razor.Logging;
+using Microsoft.CodeAnalysis.Text;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+
+namespace Microsoft.CodeAnalysis.Razor.Formatting.New;
```

Comment:
I'm really hoping you come with a cool project name instead of `New` if we have to keep both around. May I present a few options: 

1. Fazor
2. Aluminium
3. TheWengierEngine
4. Better